### DISCUSSION TITLE
ponsorship Request for Team Plan [ Open Source policy ]

### DISCUSSION THREAD

**Discussion by vzrenggamani** [vercel/community#5191] [Help] [answered]
### Summary

Team ID: team_bLYicsP95hrWobQoIAyoHCY0
Team URL: https://vercel.com/teams/gdsc-um
Project Name: Google Developer Students Universitas Negeri Malang
Project Repository: https://github.com/gdsc-um/homesite
Project Domain: https://www.gdsc-um.org/

### Steps to Reproduce

-

**Comment by swarnava** [comment]
Our open-source sponsorship program is currently paused as we make improvements.

We’re proud to have supported over 700 teams and individuals so far with almost $2M dollars of Vercel credits. This includes popular open-source projects like Tailwind CSS, Nuxt, Svelte, styled-components, as well as non-profits like the Red Cross.

While we aren’t accepting new team sponsorships at the moment, individuals can still get started building on Vercel completely free. See the [free tier usage guidelines](https://vercel.com/docs/limits/fair-use-guidelines#typical-monthly-usage-guidelines) to learn more. Thank you.

**Reply by vzrenggamani** [reply, marked as answer]
Glad to hear the current status of the program. but could I get little upgrade just for deployment from a organization? No need fo extra bandwidth or the other. Just deployment from orgs is we need @swarnava
